astropy/coordinates/solar_system.py

```diff
@@ -0,0 +1,204 @@
+                                        cartesian_position_earth,
+                                        unit=u.km)
+
+    return earth_to_planet_vector, earth_distance
+
+
+def _get_apparent_planet_position(time, planet_index):
+    """
+    Calculate the apparent position of planet ``planet_index`` in cartesian
+    coordinates, given the approximate light travel time to the object.
+
+    Uses ``jplephem`` with the DE430 kernel.
+
+    Parameters
+    ----------
+    time : `~astropy.time.Time`
+        Time of observation
+
+    planet_index : int
+        Index of the planet (1-9 for Mercury through Pluto)
+
+    Returns
+    -------
+    cartesian_position : `~astropy.coordinates.CartesianRepresentation`
+        Position of the planet defined as a vector from the Earth.
+    """
+    # Get distance of planet at `time`
+    earth_to_planet_vector, earth_distance = _get_absolute_planet_position(time, planet_index)
+
+    # The apparent position depends on the time that the light was emitted from
+    # the distant planet, so subtract off the light travel time
+    light_travel_time = earth_distance/speed_of_light
+    emitted_time = time - light_travel_time
+
+    # Calculate position given approximate light travel time.
+    # TODO: this should be solved iteratively to converge on precise positions
+    earth_to_planet_vector, earth_distance = _get_absolute_planet_position(emitted_time, planet_index)
+    x, y, z = earth_to_planet_vector
+
+    return CartesianRepresentation(x=x, y=y, z=z)
+
+
+def get_planet(time, planet_index, location=None):
+    """
+    Get a `~astropy.coordinates.SkyCoord` for a planet as observed from a
+    location on Earth.
+
+    Parameters
+    ----------
+    time : `~astropy.time.Time`
+        Time of observation
+
+    planet_index : int
+        Index of the planet (1-9 for Mercury through Pluto), excluding the
+        special value 3, which corresponds to the Earth's Moon.
+
+    location : `~astropy.coordinates.EarthLocation`
+        Location of observer on the Earth. If none is supplied, set to
+        Greenwich.
+
+    Returns
+    -------
+    skycoord : `~astropy.coordinates.SkyCoord`
+        Coordinate for the planet
+    """
+    if location is None:
+        location = EarthLocation.of_site('greenwich')
+
+    cartrep = _get_apparent_planet_position(time, planet_index)
+
+    return SkyCoord(GCRS(cartrep, obstime=time,
+                         obsgeoloc=u.Quantity(location.geocentric, copy=False)))
+
+
+def get_moon(time, location=None):
+    """
+    Get a `~astropy.coordinates.SkyCoord` for the Earth's moon as observed
+    from a location on Earth.
+
+    Parameters
+    ----------
+    time : `~astropy.time.Time`
+        Time of observation
+
+    location : `~astropy.coordinates.EarthLocation`
+        Location of observer on the Earth. If none is supplied, set to
+        Greenwich.
+
+    planet_index : int
+        Index of the planet (1-9 for Mercury through Pluto)
```

Comment:
I thought about this and decided that it's still pretty clear what `9` represents. Does anyone think that pluto should be inaccessible from the `get_planet` function?